frameworks/Rust/salvo/benchmark_config.json

```diff
@@ -47,6 +46,39 @@
         "fortune_url": "/fortunes",
         "query_url": "/queries?q=",
         "update_url": "/updates?q=",
+        "port": 8080,
+        "approach": "Realistic",
+        "classification": "Micro",
+        "database": "Postgres",
+        "framework": "salvo",
+        "language": "Rust",
+        "orm": "Raw",
+        "platform": "Rust",
+        "webserver": "Hyper",
+        "os": "Linux",
+        "database_os": "Linux",
+        "display_name": "Salvo [PG]",
+        "notes": "",
+        "versus": ""
+      },
+      "cache-diesel": {
```

Comment:
Is there a particular reason to introduce permutations specifically for the cached queries test instead of expanding `diesel` and `pg`? That is increasing the runtime quite a lot.